src/components/subscriptions/SubscriptionUsageLifetimeGraph.tsx

```diff
@@ -111,14 +111,14 @@ const SubscriptionUsageLifetimeGraph = ({
         alignItems={'flex-start'}
         justifyContent={'space-between'}
       >
-        <Stack direction={'row'} gap={2} alignItems={'flex-end'}>
+        <div className="align flex flex-row items-center gap-2">
```

Comment:
what's align?

Reply:
Yeah took me tries to find the good one, this one does not exists actually

Does a linter exists to flag those non existing rules?